curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc23991f86c76b9248c6eb8.md

```diff
@@ -33,7 +33,7 @@ assert(
 Your `h2` element should be below the `main` element's opening tag and its opening tag should start 6 spaces over from the start of the line.
 
 ```js
-assert(code.toLowerCase().match(/<main\>\n\s{6}<h2>/));
+assert(code.toLowerCase().match(/<main\>\s*\n\s{6}<h2>/));
```

Comment:
```suggestion
assert(code.toLowerCase().match(/<main\>\s*\n\s{6,}<h2>/))
```

Reply:
That would allow having uneven indentation for the part of code that should be indented in this step. Is that the intent?

Reply:
I like its look with 7 space more than 6😅. 

But for the people who understood the intent of the step, they don't have to sit there doing (1, 2, 3, 4, 5, 6), yeab these are 6 spaces. I remember doing this at this step, and it was so redundant that I skipped the other indenting step. 

Note: This isn't a project, so I am totally fine with it staying the same way because it doesn't affect the certification

Reply:
oh you meant something like this 

```html 
 <main>
           <h2>Cat Photos</h2>
               <!-- TODO: Add link to cat photos -->
                   <p>Click here to view more cat photos.</p>
    </main>
```

to be honest, I haven't thought of "If someone goes out of his way and ignore the example and the two more spaces instructions" but your worry is valid, I don't know how to fix it😓

Reply:
Yeah, something like:
```js
<main>
           <h2>Cat Photos</h2>
  <!-- TODO: Add link to cat photos -->
  <p>Click here to view more cat photos.</p>
</main>
```
what makes the step obsolete as a whole. And as it is, with changed only first check, indenting other lines to the same point as line with h2, will fail. I think if there's such step it won't hurt to forcing the usual JS indentation convention - 2 spaces per level.